common/certificate/src/lib.rs

```diff
@@ -86,7 +86,7 @@ impl PemCertificate {
 }
 
 pub struct KeyCertPair {
-    certificate: rcgen::Certificate,
+    certificate: Zeroizing<rcgen::Certificate>,
```

Comment:
We inadvertently moved away from of version of `rcgen` supporting zeroizing, because of this bug. The certificate __must__ be zeroized on drop, i.e be wrapped into a `Zeroizing` struct.